src/quo2/components/bottom_sheet/view.cljs

```diff
@@ -0,0 +1,218 @@
+(ns quo2.components.bottom-sheet.view
+  (:require [cljs-bean.core :as bean]
+            [quo.animated :as animated]
+            [quo.components.safe-area :as safe-area]
+            [quo.gesture-handler :as gesture-handler]
+            [quo.platform :as platform]
+            [quo.react :as react]
+            [quo.react-native :as rn]
```

Comment:
instead react and rn only rn should be used from react-native root package , same fro platform